### PR TITLE
03-1563: Use the new Actionable notification added to the core framework.

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

Use the `showActionableNotification` that was recently added to core.
Remove the `Actionable Notification Component` from patient management.

## Screenshots

Screenshot 2022-10-27 at 16 14 23

Screenshot 2022-10-27 at 15 35 45

## Related Issue

*None.*
[Use the new Actionable notification added to the core framework.](https://issues.openmrs.org/browse/O3-1563)

### CODE REVIEW

File: packages/esm-appointments-app/src/home-appointments/common.ts [hunk 1]
```diff
@@ -14,14 +9,11 @@ export const launchCheckInAppointmentModal = (appointmentUuid: string) => {
...
-  onClickUndo();
   const abortController = new AbortController();
   const { status } = await undoAppointmentStatus(appointmentUuid, abortController);
   if (status === 200) {
-    closeActionableNotification();
     mutate(`/ws/rest/v1/appointment/appointment/all`);
   } else {
-    closeActionableNotification();
     showNotification({
       title: 'Error Undoing',
```
Comment (vasharma05) [inline packages/esm-appointments-app/src/home-appointments/common.ts:18]: Is this a valid text?

File: packages/esm-appointments-app/src/home-appointments/common.ts [hunk 2]
```diff
@@ -48,7 +41,8 @@ export const handleUpdateStatus = async (
       kind: 'success',
       subtitle: successDescription,
       title: successTitle,
-      actionButtonLabel: 'Undo',
+      actionButtonLabel: t('undo', 'Undo'),
+      progressActionLabel: t('revertingAppointmentStatus', 'Reverting Appointment status'),
```
Comment (vasharma05) [inline packages/esm-appointments-app/src/home-appointments/common.ts:45]:
```suggestion
      progressActionLabel: t('revertingAppointmentStatus', 'Reverting appointment status'),
```

**Review by vasharma05 — APPROVED** [review]

**Review by vasharma05 — APPROVED** [review]
A few changes commented.
Good work!

Thanks!

File: packages/esm-appointments-app/src/home-appointments/common.ts (hunk 1 above)
Comment (hadijahkyampeire) [reply packages/esm-appointments-app/src/home-appointments/common.ts:18]: Yes there is also the `InlineNotofication` and that's what we use for errors.

**Pull request merged**